canonical/canonicalwebteam.blog
# Set categories to None and not empty list when not provided

## Done

Set categories to None and not empty list when not provided

## QA

- Go to https://ubuntu.com/blog/people-and-culture and see it errors
- Locally change `requirements.txt` of ubuntu.com project to: 
```
# canonicalwebteam.blog==6.4.0
canonicalwebteam.blog @ git+https://github.com/anthonydillon/canonicalwebteam.blog@null-no-categories
```
- Run `dotrun clean && dotrun`
- Go to http://0.0.0.0:8001/blog/people-and-culture
- See that the page loads
- Check a few groups all load correctly
